Shared/Prefs.swift

```diff
@@ -74,6 +74,7 @@ public struct PrefsKeys {
     public struct FeatureFlags {
         public static let ASPocketStories = "ASPocketStoriesUserPrefsKey"
         public static let ASSponsoredPocketStories = "ASSponsoredPocketStoriesUserPrefsKey"
+        public static let AutopushFeature = "AutopushFeatureUserPrefKey"
```

Comment:
Creating a user feature flag here because the nimbus feature flags can't be accessed from the Notification Service yet, the notification service will be checking the pref directly using `profile.prefs.boolForKey`

Reply:
The user feature flag is normally tied to a settings in the UI I think 🤔 Can we use it like this @adudenamedruby ?

Reply:
It's not ideal but we need to update how we manage feature flags to better handle this type of scenario. I think that is beyond the scope of the push refactor work tho, so this is an acceptable work around for now in my opinion.

I'll sync with Roux about it, maybe we can work something into the feature flag clean up project that's coming up.